source/js/interview.js

```diff
@@ -1,277 +1,258 @@
+/* eslint-disable linebreak-style */
```

Comment:
🙈 what's the point of having ESLint if you ignore all the rules?!

Reply:
Some of them are very annoying and not useful at all, I've created global rule set where I turned off the unnecessary rules.